index.bs

```diff
@@ -1318,23 +1340,84 @@ Whenever a [=WebTransport stream=] associated with a {{SendStream}} |stream| get
 
 </div>
 
-# `ReceiveStream` #  {#receive-stream}
+## `WebTransportSendStreamStats` Dictionary ##  {#send-stream-stats}
 
-In this spec, we call a {{ReadableStream}} providing incoming streaming features with an [=incoming
-unidirectional=] or [=bidirectional=] [=WebTransport stream=] a <dfn interface>ReceiveStream</dfn>.
+The <dfn dictionary>WebTransportSendStreamStats</dfn> dictionary includes information
+on stats specific to one {{WebTransportSendStream}}.
+
+<pre class="idl">
+dictionary WebTransportSendStreamStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long amountWrittenTo;
```

Comment:
bytesWritten

Reply:
That's what we proposed initially, which [was found confusing](https://github.com/w3c/webtransport/issues/372#issuecomment-1015538839).

My theory is the confusion stems from the implied _subject_ vs _object_ difference. E.g.
* With `(await stream.getStats()).bytesSent` // How many bytes did this _stream_ send? stream = subject
* With `(await stream.getStats()).bytesWritten` // How many bytes were written to this _stream_? stream = object

The hope was that the following would be clearer:
* With `(await stream.getStats()).amountWrittenTo` // How many bytes were written to this _stream_?

Disclaimer: there was some [pushback](https://github.com/w3c/webtransport/issues/372#issuecomment-1040430215) on "*To" and "*From", but it wasn't voiced at last meeting.